tests/unit/test_results.py

```diff
@@ -0,0 +1,139 @@
+import copy
+import datetime
+import random
+
+import responses
+
+from intezer_sdk.analyses_history import general_data_analyses_history
+from intezer_sdk.api import get_global_api
+from intezer_sdk.results import Results
+from tests.unit.base_test import BaseTest
+
+FILE_ANALYSES_REQUEST = '/analyses/history'
```

Comment:
maybe just move this to api module

Reply:
Don't understand why? It's been use only at this module